### DISCUSSION TITLE
divider in vertical mode

### DISCUSSION THREAD

**Discussion by proton029** [Budibase/budibase#6660] [Help] [answered]
I have been trying to put a divider in vertical mode between two containers arranged side by side. But the divider seems be showing like a dot in vertical mode instead of a vertical line. I have tried rotating 90 degrees in CSS for  a horizontal divider, that works but its hard to adjust it to the position.

![image](https://user-images.githubusercontent.com/88423620/178466998-562dd389-3897-488a-a7d2-de29f02c216b.png)

**Comment by joebudi** [comment, marked as answer]
Hey @proton029 
Could you send a screenshot of your entire component tree? As you can see from the img below, it works for me.
![CleanShot 2022-07-12 at 14 02 30](https://user-images.githubusercontent.com/49767913/178496320-d17e6a3d-88bb-4351-a23c-b5224b56d46e.png)

**Reply by proton029** [reply]
@joebudi 
Hi. Thank you for reply. Basically the form has multiple steps with a single form and I added the divider in vertical format in the 5th step between two containers which are inside another container in row alignment.  I cant share a screenshot of entire screen components as it wont fit into a single screenshot. I'll share the component tree of the section where i added the divider.
![image](https://user-images.githubusercontent.com/88423620/178647097-bcd135b9-dcb3-4e24-9502-42e52513bc52.png)

**Reply by joebudi** [reply]
Cheers. A workaround might be to add height to the component in the custom css settings.

`height: 200px;`

**Reply by proton029** [reply]
@joebudi 
Yes adding custom css to it does work for me, also I've tried rotating 90 degs for a horizontal divider. It would have been less work without the css
